sources/packages/web/src/views/supporting-user/SupportingInformation.vue

```diff
@@ -8,47 +8,53 @@
       application."
     >
     </body-header>
-    <content-group class="mb-4">
-      <v-row>
-        <v-col>
-          <v-text-field
-            density="compact"
-            label="Application number"
-            variant="outlined"
-            v-model="applicationNumber"
-            data-cy="applicationNumber"
-            type="number"
-            :rules="[(v) => /\d+/.test(v) || 'Invalid application number']"
-          />
-        </v-col>
-        <v-col>
-          <v-text-field
-            density="compact"
-            label="Student's last name"
-            variant="outlined"
-            v-model="studentsLastName"
-            data-cy="studentsLastName"
-            :rules="[(v) => !!v || 'Student\'s last name required']"
-          />
-        </v-col>
-        <v-col>
-          <v-text-field
-            density="compact"
-            label="Student's date of birth"
-            variant="outlined"
-            v-model="studentsDateOfBirth"
-            data-cy="studentsDateOfBirth"
-            type="date"
-            :rules="[(v) => !!v || 'Student\'s date of birth required']"
-          />
-        </v-col>
-        <v-col
-          ><v-btn color="primary" @click="applicationSearch"
-            >Search</v-btn
-          ></v-col
-        >
-      </v-row>
-    </content-group>
+    <v-form ref="searchApplicationsForm">
+      <content-group class="mb-4">
+        <v-row>
+          <v-col>
+            <v-text-field
+              density="compact"
+              :label="applicationNumberLabel"
+              variant="outlined"
+              v-model="applicationNumber"
+              data-cy="applicationNumber"
```

Comment:
you can use `hide-details=auto`. it will save the bottom space. it was use for all input fields

Reply:
ok. thanks!